eth/v1alpha1/beacon_chain.proto

```diff
@@ -81,7 +81,7 @@ service BeaconChain {
     // 
     // This includes the head block slot and root as well as information about
     // the most recent finalized and justified slots.
-    rpc GetChainHead(google.protobuf.Empty) returns (ChainHead) {
+    rpc GetChainHead(google.protobuf.Empty) returns (stream ChainHead) {
```

Comment:
IMO, the stream needs to be an additional gRPC endpoint.

Reply:
Good idea